### ISSUE TITLE
Please update and maintain the plugin in Spigot. 

### ISSUE THREAD

**Issue by Aadi-mone** [games647/FastLogin#628]
### What happened?

Plugin is now dead 

### What did you expect?

Please update the plugin for 1.16.5 and 1.17 on the "Spigot Page"
I am on a budget so I can't buy a host, So I use aternos.
And updating and adding a version in spigot doesn't cost.
https://www.spigotmc.org/resources/fastlogin.14153/

Best Regards,
~ Adarsh A

### Steps to reproduce

Please update the plugin for 1.16.5 and 1.17 on the "Spigot Page"
I am on a budget so I can't buy a host, So I use aternos.
And updating and adding a version in spigot doesn't cost.
https://www.spigotmc.org/resources/fastlogin.14153/

Best Regards,
~ Adarsh A

### Plugin list

FastLogin, Userlogin. 

### Server log

```shell
Please update the plugin for 1.16.5 and 1.17 on the "Spigot Page"
I am on a budget so I can't buy a host, So I use aternos.
And updating and adding a version in spigot doesn't cost.
https://www.spigotmc.org/resources/fastlogin.14153/

Best Regards,
~ Adarsh A
```

### Plugin version

....

### Platform

Spigot

### Relevance

- [X] I tried the latest build
- [X] I checked for existing tickets -
If there are, please vote them with a thumps reaction and not create new ones

**Comment by games647** [comment]
This more like an enhancement request. Updating the plugin on Spigot requires more rigorous testing and preparing a changelog. It's time consuming. That's why I tried to work towards automatic testing to reduce this "overhead" in the future. Once this is finished it will be much easier.

**Comment by Aadi-mone** [comment]
so you're not planning to update it on the spigot page in short right?

**Comment by games647** [comment]
Duplicate of #478
